src/material-experimental/mdc-checkbox/checkbox.ts

```diff
@@ -46,6 +46,9 @@ import {numbers} from '@material/ripple';
 
 let nextUniqueId = 0;
 
+// Default checkbox configuration.
+const defaults = MAT_CHECKBOX_DEFAULT_OPTIONS_FACTORY();
```

Comment:
Invoking the factory directly feels a little weird, but I'm trying to avoid repeating the default color across different files.

Reply:
Yeah I'm not even sure why its a factory, feels like it could just be provided with `useValue`. I'm fine with doing this, we can revisit if it ever becomes problematic.